### PR TITLE
feat(pci.ai-dashboard): datatr-68 - landing page ai dashboard

### PR DESCRIPTION
**Pull request by jperchoc**
DATATR-68

Signed-off-by: Jonathan Perchoc <jonathan.perchoc@ovhcloud.com>

| Question         | Answer
| ---------------- | ---
| Branch?          | `feat/datatr-106-ai-dashboard-base`
| Bug fix?         | no
| New feature?     | yes
| Breaking change? |no
| Tickets          | Fix #DATATR-68
| License          | BSD 3-Clause

- [x] Try to keep pull requests small so they can be easily reviewed.
- [x] Commits are signed-off
- [x] Only FR translations have been updated
- [x] Branch is up-to-date with target branch
- [x] Lint has passed locally
- [x] Standalone app was ran and tested locally
- [x] Ticket reference is mentioned in linked commits (internal only)
- [ ] Breaking change is mentioned in relevant commits

## Description

Add the onboarding page

## Related
Release branch: https://github.com/ovh/manager/tree/release/ai-dashboard
&nbsp;&nbsp;&nbsp;&nbsp;| -[DATATR-106](https://github.com/ovh/manager/pull/8146)
&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;| - [DATATR-68](https://github.com/ovh/manager/pull/8160)
&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;| - [DATATR-69](https://github.com/ovh/manager/pull/8172)
&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;| - [DATATR-70](https://github.com/ovh/manager/pull/8213)
&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;| - [DATATR-71](https://github.com/ovh/manager/pull/8443)

### CODE REVIEW

File: packages/manager/modules/pci/src/projects/project/ai-dashboard/onboarding/onboarding.html [hunk 1]
```diff
@@ -1,3 +1,130 @@
 <div>
-    Onboarding page
+    <!-- First Row: pipeline & first time here -->
+    <div class="row my-2">
+        <!-- AI Pipeline-->
+        <div class="col">
+            <ai-pipeline
```
Comment (anooparveti) [inline packages/manager/modules/pci/src/projects/project/ai-dashboard/onboarding/onboarding.html:6]:
```suggestion
            <pci-project-ai-pipeline
```

File: packages/manager/modules/pci/src/projects/project/ai-dashboard/onboarding/translations/Messages_fr_FR.json [hunk 2]
```diff
@@ -1 +1,23 @@
-{}
+{
+  "pci_ai_dashboard_onboarding_tuto": "Tutoriels",
+  "pci_ai_dashboard_onboarding_tuto_notebook_title": "Créer un AI Notebook",
+  "pci_ai_dashboard_onboarding_tuto_notebook_description": "Lorem ipsum dolor, sit amet consectetur adipisicing elit. Eligendi saepe cumque odit.",
+  "pci_ai_dashboard_onboarding_tuto_training_title": "Créer un AI Training",
+  "pci_ai_dashboard_onboarding_tuto_training_description": "Lorem ipsum dolor, sit amet consectetur adipisicing elit. Eligendi saepe cumque odit.",
```
Comment (anooparveti) [inline packages/manager/modules/pci/src/projects/project/ai-dashboard/onboarding/translations/Messages_fr_FR.json:6]: **question:** Is it a valid description?

File: packages/manager/modules/pci/src/projects/project/ai-dashboard/onboarding/translations/Messages_fr_FR.json [hunk 3]
```diff
@@ -1 +1,23 @@
...
+  "pci_ai_dashboard_onboarding_tuto_deploy_title": "Déployer une application",
+  "pci_ai_dashboard_onboarding_tuto_deploy_description": "Lorem ipsum dolor, sit amet consectetur adipisicing elit. Eligendi saepe cumque odit.",
```
Comment (anooparveti) [inline packages/manager/modules/pci/src/projects/project/ai-dashboard/onboarding/translations/Messages_fr_FR.json:8]: **question:** Is it a valid description?

File: packages/manager/modules/pci/src/projects/project/ai-dashboard/onboarding/translations/Messages_fr_FR.json [hunk 4]
```diff
@@ -1 +1,23 @@
...
+  "pci_ai_dashboard_onboarding_tuto_ai_compare_title": "Comparer les AI Tools",
+  "pci_ai_dashboard_onboarding_tuto_ai_compare_description": "Lorem ipsum dolor, sit amet consectetur adipisicing elit. Eligendi saepe cumque odit.",
```
Comment (anooparveti) [inline packages/manager/modules/pci/src/projects/project/ai-dashboard/onboarding/translations/Messages_fr_FR.json:10]: **question:** Is it a valid description?

**Review by anooparveti — CHANGES_REQUESTED** [review]

File: packages/manager/modules/pci/src/projects/project/ai-dashboard/onboarding/onboarding.html (hunk 1 above)
Comment (jperchoc) [reply packages/manager/modules/pci/src/projects/project/ai-dashboard/onboarding/onboarding.html]: Changed: https://github.com/ovh/manager/pull/8160/commits/e3e42f50be6eb5aece2f0c8b17e28c9c2f03ccb4

File: packages/manager/modules/pci/src/projects/project/ai-dashboard/onboarding/translations/Messages_fr_FR.json (hunk 2 above)
Comment (jperchoc) [reply packages/manager/modules/pci/src/projects/project/ai-dashboard/onboarding/translations/Messages_fr_FR.json]: Good catch, should have been removed before. Done here: https://github.com/ovh/manager/pull/8160/commits/e3e42f50be6eb5aece2f0c8b17e28c9c2f03ccb4

File: packages/manager/modules/pci/src/projects/project/ai-dashboard/onboarding/onboarding.html [hunk 5]
```diff
@@ -0,0 +1,130 @@
...
+                        data-translate="pci_ai_dashboard_onboarding_benefits"
+                    ></span>
+                </div>
+                <div class="oui-box-block__body">
+                    <div class="row mb-2">
+                        <div class="col">
+                            <div class="d-flex">
+                                <span
+                                    class="icon-badge oui-icon oui-icon-cloud-hand_concept"
+                                ></span>
```
Comment (anooparveti) [inline packages/manager/modules/pci/src/projects/project/ai-dashboard/onboarding/onboarding.html:30]:
```suggestion
                                <span
                                    class="icon-badge oui-icon oui-icon-cloud-hand_concept"
                                    aria-hidden="true"
                                ></span>
```

File: packages/manager/modules/pci/src/projects/project/ai-dashboard/onboarding/onboarding.html [hunk 6]
```diff
@@ -0,0 +1,130 @@
...
+                                        data-translate="pci_ai_dashboard_onboarding_benefits_reversibility_descritpion"
+                                    ></p>
+                                </div>
+                            </div>
+                        </div>
+                        <div class="col">
+                            <div class="d-flex">
+                                <span
+                                    class="icon-badge oui-icon oui-icon-component_concept"
+                                ></span>
```
Comment (anooparveti) [inline packages/manager/modules/pci/src/projects/project/ai-dashboard/onboarding/onboarding.html:45]:
```suggestion
                                <span
                                    class="icon-badge oui-icon oui-icon-component_concept"
                                    aria-hidden="true"
                                ></span>
```

File: packages/manager/modules/pci/src/projects/project/ai-dashboard/onboarding/onboarding.html [hunk 7]
```diff
@@ -0,0 +1,130 @@
...
+                                </div>
+                            </div>
+                        </div>
+                    </div>
+                    <div class="row mb-2">
+                        <div class="col">
+                            <div class="d-flex">
+                                <span
+                                    class="icon-badge oui-icon oui-icon-cloud-padlock_concept"
+                                ></span>
```
Comment (anooparveti) [inline packages/manager/modules/pci/src/projects/project/ai-dashboard/onboarding/onboarding.html:62]:
```suggestion
                                <span
                                    class="icon-badge oui-icon oui-icon-cloud-padlock_concept"
                                    aria-hidden="true"
                                ></span>
```

File: packages/manager/modules/pci/src/projects/project/ai-dashboard/onboarding/onboarding.html [hunk 8]
```diff
@@ -0,0 +1,130 @@
...
+                                        data-translate="pci_ai_dashboard_onboarding_benefits_security_descritpion"
+                                    ></p>
+                                </div>
+                            </div>
+                        </div>
+                        <div class="col">
+                            <div class="d-flex">
+                                <span
+                                    class="icon-badge oui-icon oui-icon-page-certificate_concept"
+                                ></span>
```
Comment (anooparveti) [inline packages/manager/modules/pci/src/projects/project/ai-dashboard/onboarding/onboarding.html:77]:
```suggestion
                                <span
                                    class="icon-badge oui-icon oui-icon-page-certificate_concept"
                                    aria-hidden="true"
                                ></span>
```

File: packages/manager/modules/pci/src/projects/project/ai-dashboard/onboarding/onboarding.html [hunk 9]
```diff
@@ -0,0 +1,130 @@
...
+                                </div>
+                            </div>
+                        </div>
+                    </div>
+                    <div class="row">
+                        <div class="col-6">
+                            <div class="d-flex">
+                                <span
+                                    class="icon-badge oui-icon oui-icon-hand-world_concept"
+                                ></span>
```
Comment (anooparveti) [inline packages/manager/modules/pci/src/projects/project/ai-dashboard/onboarding/onboarding.html:94]:
```suggestion
                                <span
                                    class="icon-badge oui-icon oui-icon-hand-world_concept"
                                    aria-hidden="true"
                                ></span>
```

**Review by anooparveti — CHANGES_REQUESTED** [review]

**Review by ClaveauAlex — APPROVED** [review]

**Review by anooparveti — APPROVED** [review]

**Pull request merged**